contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1138,7 +1144,12 @@ select cast(t.typname as sys.sysname) as name
     as is_nullable
   -- CREATE TYPE ... FROM is implemented as CREATE DOMAIN in babel
   , CAST(1 as sys.bit) as is_user_defined
-  , CAST(0 as sys.bit) as is_assembly_type
+  , CASE tsql_type_name
```

Comment:
It should be tsql_base_type_name here.

Reply:
For case of CLR-UDT both will be same and tests are already there in  sys-assembly_types

Reply:
https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2322